pom.xml

```diff
@@ -8,6 +8,51 @@
     <version>1.0.0-SNAPSHOT</version>
     <packaging>pom</packaging>
 
+    <licenses>
+        <license>
+            <name>Apache License, Version 2.0</name>
+            <url>https://www.apache.org/licenses/LICENSE-2.0.txt</url>
+        </license>
+    </licenses>
+
+    <name>Strimzi - Apache Kafka on Kubernetes and OpenShift</name>
+    <description>Strimzi uses the Kubernetes operator pattern to provide a way to run an Apache Kafka cluster on
+        Kubernetes or OpenShift in various deployment configurations. The OAuth subproject implements the callbacks for
+        Kafka authentication using OAuth (based on SASL OAUTHBEARER mechanism).</description>
+    <url>https://strimzi.io/</url>
```

Comment:
Should we remove OpenShift mentions here?

Reply:
Done. This is copied from the main operators repo. We should maybe remove it there as well.

Reply:
Yes agree